playbooks/kind-integration-test-arm64/run.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-docker-ce
+    - andrewrothstein.bazel
+  vars:
+    - go_version: 1.12
+    - bazel_ver: '0.23.1'
+
+  tasks:
+    - name: git required repositories
+      shell:
+        cmd: |
+          go get sigs.k8s.io/kind
+          go get k8s.io/kubeadm
+          go get k8s.io/kubernetes
+          go get k8s.io/test-infra
+        executable: /bin/bash
+      environment: '{{ global_env }}'
+    - name: build KIND against k/k master using bazel
+      shell:
+        cmd: |
+          cd $GOPATH/src/sigs.k8s.io/kind && go install .
+          kind build base-image --image kindest/base:latest --source $GOPATH/src/sigs.k8s.io/kind/images/base --loglevel debug
+          kind build node-image --type bazel --image kindest/node:latest --base-image kindest/base:latest --kube-root=$GOPATH/src/k8s.io/kubernetes/ --loglevel debug
+        executable: /bin/bash
+      environment: '{{ global_env }}'
+
+    - name: run with kubetest
+      shell:
+        cmd: |
+          cd $GOPATH/src/k8s.io/test-infra/kubetest && go install .
+          cd $GOPATH/src/k8s.io/kubernetes && kubetest \
+            --provider=skeleton \
+            --deployment=kind \
+            --kind-binary-version=build \
+            --kind-base-image="kindest/base:latest" \
+            --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml \
+            --build=bazel \
```

Comment:
I think the ``--build=bazel`` is not neccessary as seems from https://github.com/kubernetes/test-infra/blob/master/kubetest/kind/kind.go#L241 if we don't pass anything, we will use existing node-image(which was built in line 27) passing other value will trigger a re-build of the node-image which makes the previous build useless.